From 26d3a6cc2b01da7475e79231725caf245f8c65fe Mon Sep 17 00:00:00 2001
From: Rebecca Chang Swee Fun <rebecca.chang@starfivetech.com>
Date: Fri, 14 Jan 2022 02:40:23 +0000
Subject: [PATCH] BUILD.gn: disable printing unittests

Depends on cups-config in sysroot.
Currently sysroot does not include cups-config.

Signed-off-by: Rebecca Chang Swee Fun <rebecca.chang@starfivetech.com>
---
 BUILD.gn | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/BUILD.gn b/BUILD.gn
index 73aefc2325553..f3bb841e5419a 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -282,10 +282,6 @@ group("gn_all") {
 
   deps += root_extra_deps
 
-  if (enable_basic_printing) {
-    deps += [ "//printing:printing_unittests" ]
-  }
-
   if (enable_extensions) {
     deps += [
       "//extensions:extensions_browsertests",
-- 
2.25.1

